src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/PersistedAssemblyBuilder.cs

```diff
@@ -202,5 +202,7 @@ protected override void SetCustomAttributeCore(ConstructorInfo con, ReadOnlySpan
         public override Module ManifestModule => _module ?? throw new InvalidOperationException(SR.InvalidOperation_AModuleRequired);
 
         public override AssemblyName GetName(bool copiedName) => (AssemblyName)_assemblyName.Clone();
+
+        public override bool IsDynamic => true;
```

Comment:
Why do we need this?

Reply:
It is unrelated with this PR, but saw it populated in the [missing docs](https://github.com/dotnet/runtime/issues/105982) list and thought is a public API that needs to be overriden

Reply:
I think we should delete from the ref instead. The abstract AssemblyBuilder provides this override already.